README.md

```diff
@@ -219,16 +219,18 @@ script is used by swift.org's CI to produce snapshots and can allow for one to
 locally reproduce such builds for development or distribution purposes. E.x.:
 
 ```
-  $ ./utils/build-toolchain $TOOLCHAIN_PREFIX
+  $ ./utils/build-toolchain $BUNDLE_PREFIX
 ```
 
-where ``$TOOLCHAIN_PREFIX`` is a string that will be prepended to the swift
-package name in the produced tar ball. For instance, if ``$TOOLCHAIN_PREFIX``
-was ``macOS``, the produced archive will have the name
-``swift-macOS.tar.gz``.
+where ``$BUNDLE_PREFIX`` is a string that will be prepended to the build 
+date to give the bundle identifier of the Toolchain's Info.plist. For 
```

Comment:
Nit: is there a reason why "toolchain" and "info.plist" are capitalized here?

Reply:
I’ll meet you half way.. Ive lowered the case on toolchain but isn’t the file called ``Info.plist``?